docker/CMSRucioClient/scripts/CMSRSE.py

```diff
@@ -310,12 +313,20 @@ def _get_protocol(self, proto_json, protos_json):
             for method, weight in domains['wan'].items():
                 if weight and protocol_name in PROTO_WEIGHT_TPC:
                     if method.startswith("third_party_copy"):
-                        domains['wan'][method] = PROTO_WEIGHT_TPC[protocol_name]
+                        if access == 'global-ro' and method != 'third_party_copy_read':
+                            domains['wan'][method] = 0
```

Comment:
If the access is `global-ro` instead of `global-rw`, only assigns priority number to `read` method. Others (`delete` and `write` is set to 0).

Reply:
This is for TPC